vickev/howdypix
# Add cypress tests

## Description

Adds the first Cypress test using mock data.





## How to test?

- Check-out this branch.
- Start the webapp: `cd apps/webapp ; NODE_ENV=test yarn dev`
- Run the tests: `yarn test:e2e:local`

## Checklist
